MarsQACompetitionTaskNUnit/Tests/CertificationTest.cs

```diff
@@ -0,0 +1,144 @@
+﻿using MarsQACompetitionTaskNUnit.Assertions;
+using NUnit.Framework;
+
+namespace MarsQACompetitionTaskNUnit.Tests
+{
+    [Parallelizable]
+    [TestFixture]
+    public class CertificationTest: BaseTest 
+    {
+        [SetUp]
+        public void SetUpCertification()
+        {           
+            certificationPageObject.NavigateToCertificationTab();
+            Thread.Sleep(1000);
+        }
+
+        [Test, Order(1), Description("This test create a new Certification record")]
+        public void TestCreateCertificationRecord()
+        {         
+            certificationPageObject.ClearCertification();
+            certificationPageObject.CreateCertificationRecord(certificationConfig[0].Certificate, certificationConfig[0].From, certificationConfig[0].Year);
```

Comment:
so future reference it is best practice to create a json file per test, that way it is easier to maintain

Reply:
Json file created per test for easier to maintain.